### ISSUE TITLE
Some submit scripts do not use the conda environment

### ISSUE THREAD

**Issue by SamBryce-Smith** [frattalab/rna_seq_single_steps#3]
This definitely applies to `submit_samtools_stats.sh` & `submit_sort_pull.sh` but possible others too.

The snakemake commands don't include `--use-conda` as so rely on dependencies being available in $PATH for the pipeline to run. Since both of these just use samtools this is somewhat likely (at least in my case), but this is a bug waiting to happen and defeats the point of having a conda environment.

I suggest also adding a `--conda-prefix` call to our general conda env space to save every new user from doing a local install.

TL;DR: update the submit scripts to ensure they include `--use-conda` and `--conda-prefix "/<path/to/general/conda_env_storage/"`
